src/charm.py

```diff
@@ -286,61 +326,32 @@ def _ensure_service_health(self) -> None:
             raise
 
     def _get_runner_manager(
-        self, token: Optional[str] = None, path: Optional[str] = None
+        self, token: str | None = None, path: GithubPath | None = None
     ) -> RunnerManager:
-        """Get a RunnerManager instance, or None if missing config.
-
-        Args:
-            token: GitHub personal access token to manager the runners with.
-            path: GitHub repository path in the format '<org>/<repo>', or the GitHub organization
-                name.
+        """Get a RunnerManager instance.
 
         Returns:
             An instance of RunnerManager.
         """
```

Comment:
The argument docstring seems to be removed, was this intended?

Reply:
Re-added. This args was removed then added back during the refactor.